consent-ui/src/app/ais/password-input-page/password-input-page.component.html

```diff
@@ -1 +1,63 @@
-<p>password-input-page works!</p>
+<div class="d-md-flex align-items-center h-100 align-items-stretch">
+
+  <!-- Logo -->
+  <div class="bg-sm-light sticky-top">
+    <nav class="navbar navbar-light text-primary py-3">
+      <a class="navbar-brand" href="#">
+        <img class="logo" [src]="" />
```

Comment:
It can be `<img class="logo" [src]="customizeService.getLogo()" />`

Reply:
resolved